src/styles/app.css

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+.App {
+  height: 100vh;
```

Comment:
This height property is causing your nasa logo to warp at different screen heights, you should take this out so it stays uniform